docs/source/guides/writer/chapters/integrating.rst

```diff
@@ -0,0 +1,45 @@
+Integrating Avocado
+===================
+
+Coverage.py
+~~~~~~~~~~~
+
+Testing software is important, but knowing the effectiveness of the tests,
+like which parts are being exercised by the tests, may help develop new tests.
+
+`Coverage.py`_ is a tool designed for measuring code coverage of Python
+programs. It runs monitoring the program's source, taking notes of which
+parts of the code have been executed.
+
+It is possible to use Coverage.py while running Avocado tests. As Avocado
```

Comment:
I know that you have told that this is Python programs before, but would be nice to state again that this is only valid for Instrumented Tests here, since Avocado tests is too generic.

Reply:
That is true; I'll update that.